data/presets/presets/place/welcome_sign.json

```diff
@@ -0,0 +1,15 @@
+{
+  "icon": "maki-lighthouse",
```

Comment:
Could you find an icon that looks more like a sign? Or pick a generic icon if you can't find one?

Reply:
I found this hold it fits: 
![sign](https://user-images.githubusercontent.com/33323077/93399606-cfd61f80-f875-11ea-8571-df9a0104ac5b.png)

Reply:
I found this, I hope it fits: 
![sign](https://user-images.githubusercontent.com/33323077/93399606-cfd61f80-f875-11ea-8571-df9a0104ac5b.png)